client/src/components/shared/Layout/LayoutConnectWallet/LayoutConnectWallet.scss

```diff
@@ -0,0 +1,116 @@
+// Custom styles for RainbowKit connect modal.
+// !important are required, since most of these styles are being overwritten by direct CSS rules.
+
+.rainbowKitCustomFontFamily {
+  font-family: 'Inter', sans-serif !important;
+}
+
+.modalWidth {
+  width: $modalDesktopWidth !important;
+}
+
+.primaryWrapper {
+  padding: $modalPadding !important;
+  margin: 0 !important;
+}
+
+.modalHeader {
+  line-height: $modalHeaderHeight !important;
+  height: $modalHeaderHeight !important;
+  align-items: center !important;
+}
+
+.dummyDiv {
+  display: none !important;
+}
+
+.headerTextWrapper {
+  padding: 0 !important;
+}
+
+.headerText {
+  font-size: $modalHeaderFontSize !important;
+  font-weight: $font-weight-bold !important;
+}
+
+.optionsWrapper {
+  padding: 0 !important;
+  margin: 0 !important;
+}
+
+.walletOptionsButtonsWrapper {
+  display: flex !important;
+  flex-direction: row !important;
+}
+
+.walletOptionsButtons {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  width: 100% !important;
+  gap: 1.6rem !important;
+}
+
+.walletOptionsButtonsContent {
+  height: 9.6rem;
+  padding: 0 !important;
+  background: $color-octant-grey3 !important;
```

Comment:
When hovering over `. walletOptionsButtonsContent`, filter value for `. walletOptionsIcons` should be `null` to display the coloured logo.

```
  &:hover {
    .walletOptionsIcons {
      filter: none !important;
    }
  }
```